.github/workflows/CI.yml

```
@@ -9,7 +9,7 @@ on:
 jobs:
   test:
     runs-on: ${{ matrix.os }}
-    continue-on-error: ${{ matrix.version == 'nightly' }}
+    continue-on-error: ${{ matrix.version == 'nightly' || matrix.os == 'ubuntu-latest' }}
```

This should be reverted as well before the PR is merged (I assumed it was changed for debugging):
```suggestion
    continue-on-error: ${{ matrix.version == 'nightly' }}
```

---

Done :+1: